web/src/popover_menus_data.ts

```diff
@@ -11,6 +11,7 @@ import * as hash_util from "./hash_util";
 import {$t} from "./i18n";
 import * as message_edit from "./message_edit";
 import * as message_lists from "./message_lists";
+import {get_messages_in_topic} from "./message_util";
```

Comment:
Please use [namespace imports](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#namespace_import), instead of [named imports](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#named_import) — it helps with the code readability.

Reply:
I apologize for the inconvenience caused and will remember this in the future. Thank you.